Playbooks/playbook-Endpoint_Enrichment_-_Cylance_Protect_v2.yml

```diff
@@ -0,0 +1,317 @@
+id: Endpoint Enrichment - Cylance Protect v2
+version: -1
+fromversion: 4.1.0
+name: Endpoint Enrichment - Cylance Protect v2
+description: Enriches endpoints using the Cylance Protect v2 integration.
+starttaskid: "0"
+tasks:
+  "0":
+    id: "0"
+    taskid: 1635626d-e92f-49c7-85a7-b4d1632d1e38
+    type: start
+    task:
+      id: 1635626d-e92f-49c7-85a7-b4d1632d1e38
+      version: -1
+      name: ""
+      description: ""
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "1"
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 450,
+          "y": 50
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+  "1":
+    id: "1"
+    taskid: 50264655-eac1-4439-809b-32f93a19c825
+    type: condition
+    task:
+      id: 50264655-eac1-4439-809b-32f93a19c825
+      version: -1
+      name: Is Cylance Protect v2 enabled?
+      description: Checks if there is any active instance of the Cylance Protect v2
```

Comment:
Checks if there is an active instance of the Cylance Protect v2 integration enabled.